src/server/licence/postExitPage/index.njk

```diff
@@ -9,6 +9,7 @@
         <p>
           If you want to receive your licence by post, you need to
           <a
+            data-testid="complete-application-link"
```

Comment:
maybe rename this to complete-paper-application-link, I got super confused by that name in the tests